### DISCUSSION TITLE
Fast Login Bungeecord Not Work

### DISCUSSION THREAD

**Discussion by NMattyy** [games647/FastLogin#605] [Q&A] [answered]
Hi I can't set up FastLogin with Bungeecord
I follow the guide but every time it says sending request without doing anything
As a second authentication plugin I use AuthMe

**Comment by TuxCoding** [comment, marked as answer]
Please post the bungee server log.

**Comment by vaporvee** [comment]


**Reply by Smart123s** [reply]
> [17:59:42] [main/INFO]: Enabled Waterfall version git:Waterfall-Bootstrap:1.17-R0.1-SNAPSHOT:93773f9:448 
[...]
> [17:59:42] [main/WARN]: Error loading plugin ProtocolLib
java.lang.NoClassDefFoundError: org/<ins>bukkit</ins>/plugin/java/JavaPlugin

You don't need ProtocolLib for Waterfall. It's Bukkit only. FastLoginBungee doesn't require a protocol plugin.

----------

> 	[17:59:44] [main/WARN]: Exception encountered when loading plugin: FastLogin
fastlogin.hikari.pool.HikariPool$PoolInitializationException: Failed to initialize pool: Communications link failure 
[...]
Caused by: java.net.ConnectException: Connection refused 

It seems like Waterfall is unable to communicate with your Paper server via plugin messages. Did you enable `bungeecord=true` in the `spigot.yml` file of your paper server?

**Comment by Deeptanil** [comment]
Same issue here, I'm not sure what I'm doing wrong...
I've MySQL for authme, do i add MySQL to fastlogin?? If so, do i use the same database?

**Reply by TuxCoding** [reply]
No they don't need the same database, but you still need to set it up for FastLogin.

**Reply by Deeptanil** [reply]
What does it do?

**Reply by TuxCoding** [reply]
FastLogin needs to save the premium state somewhere.

**Reply by Deeptanil** [reply]
I've opened another question #730 let's take it there!
